composer.json

```diff
@@ -1,46 +1,53 @@
 {
-	"name": "phpab/phpab",
-	"type": "library",
-	"description": "A PHP library to create A/B tests.",
-	"keywords": [
-		"Phpab",
-		"Ab test",
-		"abtest"
-	],
-	"homepage": "https://github.com/phpab/phpab",
-	"license": "MIT",
+    "name": "phpab/phpab",
+    "type": "library",
+    "description": "A PHP library to create A/B tests.",
+    "keywords": [
+        "Phpab",
+        "Ab test",
+        "abtest"
+    ],
+    "homepage": "https://github.com/phpab/phpab",
+    "license": "MIT",
     "authors": [
-		{
```

Comment:
would be better to revert this and fix it in a different branch then. We're running in conflict when merging (because the other PRs (mine) changed that file, too.